### PR TITLE
Updated BC-Image repository name

### PR DESCRIPTION
**Pull request by staedter**
Docker pull fails because the repository name for the generic image was changed to mcr.microsoft.com/businesscentral

### CODE REVIEW

File: charts/business-central/values.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 image:
   # image.repository -- Repository of the image to use. You can use a generic image and set `image.artifactUrl` or use a pre-built image and set `image.artifactUrl` to `""`. See [here](#using-artifacts-and-pre-built-images) for more information. around the two options.
-  repository: mcr.microsoft.com/dynamicsnav
+  repository: mcr.microsoft.com/businesscentral
   # image.tag -- Tag of the image to use
   tag: 10.0.17763.1397-generic
```
Comment (lippertmarkus) [inline charts/business-central/values.yaml:5]: I think we need to remove `-generic` here as well according to the new naming

**Review by lippertmarkus — CHANGES_REQUESTED** [review]
Thanks for the contribution. See comments. Also please update the `version` within the `Chart.yaml`

File: charts/business-central/values.yaml (hunk 1 above)
Comment (staedter) [reply charts/business-central/values.yaml]: Yeah you were right. I removed the -generic tag

Comment (staedter) [reply charts/business-central/values.yaml]: Hm, I think I fixed all the linting errors but I don't know how to get passed Validating Maintainer 404 error.

Btw I realized that the Generic Image is to old for the win nodes OS... I added accept outdated but is there anyway we could use the Get-BestGenericImageName or something to dynamically update the generic image version to the current winver on the azure vms?

Comment (lippertmarkus) [reply charts/business-central/values.yaml]: just digged into the linting. I think the  maintainer name in the `chart.yaml` needs to be a valid GitHub name. Could you change that to `lippertmarkus` to see if that works? Strange behaviour as it did work before with my real name..

According to the generic image version: Automatic determination is not very trivial. You could just use `kubectl get node -o wide` to get the kernel version of your Windows workers and then add `--set image.tag "10.0.17763.1757"` with your kernel version.

**Review by lippertmarkus — APPROVED** [review]

**Comment by lippertmarkus** [comment]
Thanks for your contribution!

**Pull request merged**